src/styles/miscStyles.js

```diff
@@ -186,4 +186,10 @@ export default ({ color, backgroundColor, borderColor, cardColor, dividerColor }
     flexDirection: 'column',
     justifyContent: 'flex-end',
   },
+  emptyStateContainer: {
+    flex: 1,
+    padding: 16,
+    marginTop: 10,
```

Comment:
Previously the marginTop was `8`. Any good reason to change it to `10`? Also `8` is likely the correct value according to the 16/8 values we use (or maybe `16` if you need more space)

Reply:
@borisyankov Done! let's follow the 8/16 pattern. 👍